src/server/types.ts

```diff
@@ -140,8 +140,23 @@ export type StrandUpdate = {
 
 export type SyncStatus = 'SYNCING' | UpdateStatus;
 
+export type PullSyncStatus = 'INITIAL_SYNC' | SyncStatus;
+export type PushSyncStatus = 'INITIAL_SYNC' | SyncStatus;
```

Comment:
Although this 2 types are the same, I wanted to keep them separated, just in case we want to add different states for them in the future

Reply:
Should the aggregated sync status also have the INITIAL_SYNC?